src/Kovan-DssSpell.sol

```diff
@@ -16,36 +16,283 @@
+    //
+    uint256 constant TWO_PCT            = 1000000000627937192491029810;
+    uint256 constant FOUR_PT_FIVE_PCT   = 1000000001395766281313196627;
+    uint256 constant SIX_PCT            = 1000000001847694957439350562;
+    uint256 constant SEVEN_PCT          = 1000000002145441671308778766;
 
-    uint256 constant WAD = 10**18;
-    uint256 constant RAY = 10**27;
-    uint256 constant RAD = 10**45;
+    // Math
+    uint256 constant THOUSAND = 10 ** 3;
+    uint256 constant MILLION  = 10 ** 6;
+    uint256 constant WAD      = 10 ** 18;
+    uint256 constant RAY      = 10 ** 27;
+    uint256 constant RAD      = 10 ** 45;
 
-    address constant MCD_FLASH = 0x5aA1323f61D679E52a90120DFDA2ed1A76E4475A;
+    // Maker changelog 
+    address public constant MAKER_CHANGELOG = 0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F;
 
     function actions() public override {
-        // ---------------------------- Add MCD_FLASH ----------------------------
-        address MCD_VAT = DssExecLib.vat();
-        Fileable(MCD_FLASH).file("max", 500_000_000 * WAD);
-        Fileable(MCD_FLASH).file("toll", 5 * WAD / 10000);
-        DssExecLib.authorize(MCD_VAT, MCD_FLASH);
-        DssExecLib.setChangelogAddress("MCD_FLASH", MCD_FLASH);
-
-        // ---------------------------- Update Chainlog version ----------------------------
-        DssExecLib.setChangelogVersion("1.9.1");
+        // https://vote.makerdao.com/polling/QmRCn7Mh#poll-detail
+        CentrifugeCollateralValues memory RWA003 = CentrifugeCollateralValues({
+            MCD_JOIN: 0x4CCc7fED3912A32B6Cf7Db2FdA1554a9FF574099,
+            GEM: 0xDBC559F5058E593981C48f4f09fA34323df42d51,
+            OPERATOR: 0x45e17E350279a2f28243983053B634897BA03b64,
+            INPUT_CONDUIT: 0x45e17E350279a2f28243983053B634897BA03b64,
+            OUTPUT_CONDUIT: 0x45e17E350279a2f28243983053B634897BA03b64,
+            URN:  0x993c239179D6858769996bcAb5989ab2DF75913F,
+            LIQ: 0x2881c5dF65A8D81e38f7636122aFb456514804CC,
+            gemID: "RWA003",
+            joinID: "MCD_JOIN_RWA003_A",
+            urnID: "RWA003_A_URN",
+            inputConduitID: "RWA003_A_INPUT_CONDUIT",
+            outputConduitID: "RWA003_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA003",
+            ilk: "RWA003-A",
+            ilkRegistryName: "RWA003-A: Centrifuge: ConsolFreight",
+            RATE: SIX_PCT,
+            CEIL: 2 * MILLION,
+            PRICE: 2_359_560 * WAD,
+            MAT: 10_500,
+            TAU: 0,
+            DOC: "QmQMNfSbGS8qkJbatQgxMUsz27G8YELWgtXeLs8uFCZoY8"
+        });
+
+        // https://vote.makerdao.com/polling/QmRWg6nZ#poll-detail
+        CentrifugeCollateralValues memory RWA004 = CentrifugeCollateralValues({
+            MCD_JOIN: 0xa92D4082BabF785Ba02f9C419509B7d08f2ef271,
+            GEM: 0x146b0abaB80a60Bfa3b4fDDb5056bBcFa4f1fec1,
+            OPERATOR: 0x303dFE04Be5731207c5213FbB54488B3aD9B9FE3,
```

Comment:
This is pointing to the urn of RWA005 (0xdB9f0700EbBac596CCeF5b14D5e23664Db2A184f) instead of the one in this config (0xf22C7F5A2AecE1E85263e3cec522BDCD3e392B59).

Reply:
My bad, I set the wrong one this morning. Just fixed it. Note that for mainnet, the `urn`s will still be unset (`0x0`) on the manager contracts, as we are filing these after the maker spell has passed, in the integration spells that we're doing on our side.